phpxmlrpc/phpxmlrpc/2022-11-28-1.yaml

```diff
@@ -0,0 +1,8 @@
+title:     local file access in `Client:send` via manipulation of `$protocol` argument
+link:      https://github.com/gggeek/phpxmlrpc-extras/releases/tag/4.9.0
```

Comment:
This link is a 404.

Taking the composer package name and looking on Packagist, it looks like the right link should be https://github.com/gggeek/phpxmlrpc/releases/tag/4.9.0 (no `-extras` in the repo name).

Same for the other advisory.

Reply:
doh! copypasta

Reply:
Fixed in 2nd commit